<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="MainLayout_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="MainLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="color">
         <item>
          <widget class="QRadioButton" name="red">
           <property name="text">
            <string>Red</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="green">
           <property name="text">
            <string>Green</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="blue">
           <property name="text">
            <string>Blue</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="style">
         <item>
          <widget class="QRadioButton" name="solid">
           <property name="text">
            <string>Solidline</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="dash">
           <property name="text">
            <string>Dashline</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="dot">
           <property name="text">
            <string>Dotline</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="thickness">
         <item>
          <widget class="QRadioButton" name="thin">
           <property name="text">
            <string>Thin</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="normal">
           <property name="text">
            <string>Normal</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="thick">
           <property name="text">
            <string>Thick</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="shape">
         <item>
          <widget class="QRadioButton" name="line">
           <property name="text">
            <string>Line</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rectangle">
           <property name="text">
            <string>Rectangle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ellipse">
           <property name="text">
            <string>Ellipse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>563</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="draw">
       <property name="text">
        <string>Draw</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="select">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="move">
       <property name="text">
        <string>Move</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="scale">
       <property name="text">
        <string>Scale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider2">
       <property name="minimum">
        <number>-25</number>
       </property>
       <property name="maximum">
        <number>25</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ChangeAttributes">
       <property name="text">
        <string>ChangeAttributes</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
